steel-research-international.csl

```diff
@@ -0,0 +1,220 @@
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="year" font-weight="bold"/>
+    </date>
+    <terms>
+      <term name="chapter" form="short">Ch.</term>
+      <term name="editor" form="short">eds</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name sort-separator=" " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="book">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else-if type="chapter">
+        <text term="in" suffix=" "/>
+        <text variable="title" font-style="italic"/>
+      </else-if>
+      <else-if type="paper-conference">
+        <text term="in" suffix=" "/>
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else-if type="speech">
+        <text term="presented at" suffix=" "/>
+        <text variable="title" font-style="italic"/>
+      </else-if>
+      <else-if type="thesis">
+        <text variable="genre" font-style="italic"/>
+      </else-if>
+      <else-if type="article-journal" match="any">
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else-if type="article-magazine article-newspaper" match="any">
+        <text variable="container-title" font-style="italic"/>
+      </else-if>
+      <else-if type="webpage" match="any">
+        <text variable="title"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="editor">
+    <choose>
+      <if variable="editor">
+        <group prefix=" (" suffix=")">
+          <names variable="editor">
+            <label form="short" text-case="capitalize-first" suffix=": "/>
+            <name sort-separator=" " initialize-with="." delimiter=", " delimiter-precedes-last="always"/>
+          </names>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper patent report" match="none">
+        <group delimiter=", " prefix=", ">
+          <text variable="publisher"/>
+          <choose>
+            <!-- Specifications say if not obvious ... -->
+            <if type="thesis" match="none">
+              <text variable="publisher-place"/>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage" match="any">
+        <choose>
+          <if variable="URL">
+            <group delimiter=" " prefix=", ">
+              <text variable="URL"/>
+            </group>
+            <text macro="accessed-date"/>
+          </if>
+        </choose>
+      </if>
+      <else-if variable="DOI">
+        <group font-style="normal" text-decoration="none">
+          <text variable="DOI" prefix=", "/>
```

Comment:
No prefix like doi: or https://doi.org?

Reply:
From the template I think `DOI:` should be used.